### PR TITLE
Add label visualization

### PR DESCRIPTION
**Pull request by fyu**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -72,5 +70,9 @@
   "eslint.options": {
     "configFile": "./.eslintrc.json"
   },
-  "eslint.validate": ["typescript", "typescriptreact"]
+"eslint.validate": [
+    "typescript",
+    "typescriptreact"
+],
+"python.pythonPath": "/Users/suntao/anaconda3/envs/py38/bin/python"
```
Comment (fyu) [inline .vscode/settings.json:77]: @suniique make sure not to add your personal information in the commit.

Comment (suniique) [reply .vscode/settings.json:77]: OK, fixed.

File: scalabel/label/io_test.py [hunk 2]
```diff
@@ -1,53 +1,70 @@
...
-    labels = load(filepath)
-    labels_dict = dump(labels)
-    assert "box3d" not in labels_dict[0]["labels"][0]
-    assert "box2d" in labels_dict[0]["labels"][0]
-    assert labels[0].labels[0].box_2d is not None
-    assert (
-        labels_dict[0]["labels"][0]["box2d"]["x1"]
-        == labels[0].labels[0].box_2d.x1
-    )
+class TestIO(unittest.TestCase):
```
Comment (fyu) [inline scalabel/label/io_test.py:8]: the original `pytest` syntax is more concise, especially for `None` check

Comment (suniique) [reply scalabel/label/io_test.py]: Yes, I switched back.

**Comment by suniique** [comment]
**Command:**

  - For local images
  ```bash
  python3 -m scalabel.vis.labels --labels {path to your label file (.json)} --image-dir {path to your images folder}
  ```

  - For remote images (read from the label file's `url` fields)
  ```bash
  python3 -m scalabel.vis.labels --labels {path to your label file (.json)}
  ```
  
  - Save the visualization results as images
  ```bash
  python3 -m scalabel.vis.labels ...... --output_dir {path to output dir}
  ```

**Keymap:**
  -  N / P: Show next or previous image
  -  Space: Start / stop animation
  -  T: Toggle 2D / 3D bounding box (if avaliable)
  -  Y: Toggle image / segmentation view (if avaliable)

**Screenshot:**
screen_shot1

**Pull request merged**